databases/week1/week1.sql

```diff
@@ -0,0 +1,60 @@
+--Find out how many tasks are in the task table
+SELECT COUNT(id) as total_task FROM task;
+
+--Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(id)
+FROM task
+WHERE due_date IS NULL;
```

Comment:
General tip, a field can be invalid but not be null, for example, empty strings like '' wouldn't be captured with this expression.